### DISCUSSION TITLE
【Move 0x03】通过 React dAPP 成功进行本地的函数调用

### DISCUSSION THREAD

**Discussion by 99Kies** [NonceGeek/Move-Camp#113] [Projects-Others]
教程见：

https://github.com/WeLightProject/Web3-dApp-Camp/blob/main/move-dapp/CryptoXiuXian%20in%20Move%20Lang.md#14-your-first-move-dapp--starcoin-dapp

回复截图打卡

**截图场景一**

![image](https://user-images.githubusercontent.com/45918704/181748568-a53d01aa-2c2d-4008-87e5-444c1eb74b92.png)

**截图场景二**

![image](https://user-images.githubusercontent.com/45918704/181748634-eff5cc4a-df5c-44fb-93b8-2e97673dd152.png)

**Comment by qiwihui** [comment]
haha

![Screen Shot 2022-07-29 at 5 37 06 PM](https://user-images.githubusercontent.com/3297411/181751485-62cc54e0-76b7-402c-aafd-734c41ee053a.png)
![Screen Shot 2022-07-29 at 5 37 22 PM](https://user-images.githubusercontent.com/3297411/181751488-f7c098ef-4391-4030-9606-57fcb95de4af.png)

**Reply by leeduckgo** [reply]
Well Done! ¥88.88

**Comment by leeduckgo** [comment]
Done!

**Comment by pochenai** [comment]
<img width="805" alt="截屏2022-07-29 下午8 49 56" src="https://user-images.githubusercontent.com/13568446/181768553-9a3b0dea-299c-453e-8c6d-b73c17ca3074.png">
<img width="933" alt="截屏2022-07-29 下午9 16 20" src="https://user-images.githubusercontent.com/13568446/181768559-0c7e6b8d-4560-4c4a-94ab-7cd5c43a06e5.png">

**Comment by 0xKT** [comment]
<img width="996" alt="image" src="https://user-images.githubusercontent.com/74288668/182558358-1e55d12e-4d7a-4df5-a648-73ae3f40aa00.png">

**Comment by 99Kies** [comment]
![image](https://user-images.githubusercontent.com/45918704/184781024-b0e3aaa3-f139-4173-af24-2fe063956bf0.png)

![image](https://user-images.githubusercontent.com/45918704/184781065-fa95f633-a9b3-4e96-942f-d628edd056c9.png)

**Comment by sharkqwy** [comment]
<img width="1167" alt="截屏2022-10-17 18 52 15" src="https://user-images.githubusercontent.com/1372795/196159876-192da6e2-8fd3-4d0a-898d-499d53b8b2aa.png">

**Comment by zhaoge5859** [comment]
打卡
<img width="732" alt="截屏2022-11-01 23 27 23" src="https://user-images.githubusercontent.com/116735858/199271988-a22eb77e-fc3f-46cf-aede-39bdc57748e1.png">
<img width="377" alt="截屏2022-11-01 23 27 32" src="https://user-images.githubusercontent.com/116735858/199272012-c808b46c-2577-418a-bcd2-f2ba9b088f37.png">
<img width="369" alt="截屏2022-11-01 23 27 58" src="https://user-images.githubusercontent.com/116735858/199272023-4fadc044-1935-4d76-b6e7-e40d9b61a281.png">
<img width="330" alt="截屏2022-11-01 23 28 27" src="https://user-images.githubusercontent.com/116735858/199272026-4d306d4d-b91a-4bd7-869b-b4b4bc031bff.png">
